search-api/search-actors/src/main/java/org/sunbird/actors/SearchActor.java

```diff
@@ -767,7 +767,9 @@ private void setImplicitFilters(Map<String, Object> filters, SearchDTO searchObj
                 }
             }
             List<Map> implicitFilterProps = new ArrayList<Map>();
-            implicitFilterProps.addAll(getSearchFilterProperties(implicitFilter, false));
+            Request request = new Request();
+            request.setContext(new HashMap<String, Object>());
+            implicitFilterProps.addAll(getSearchFilterProperties(implicitFilter, false, request));
```

Comment:
Pass the null and do the null check in getSearchFilterProperties instead of passing the empty object.

Reply:
Updated suggested changes.